api/database/boltdb/json.go

```diff
@@ -2,18 +2,39 @@ package boltdb
 
 import (
 	"encoding/json"
+	"strconv"
 
 	jsoniter "github.com/json-iterator/go"
 )
 
 // MarshalObject encodes an object to binary format
 func MarshalObject(object interface{}) ([]byte, error) {
+	// Special case for the VERSION bucket. Here we're not using json
+	switch v := object.(type) {
+	case int:
+		return []byte(strconv.Itoa(v)), nil
```

Comment:
I dont think this is an issue; the marshaller can deal with int already (according to the tests)

Reply:
^ updated in the last commit - https://github.com/portainer/portainer/pull/6286/commits/f7a4a6fae5fea6842d925b93a997d17da9c1905b

Reply:
I put it in because the value was being altered from what it was before